src/oscar/apps/es_search/models.py

```diff
@@ -0,0 +1,12 @@
+from oscar.core.loading import is_model_registered
+
+from .abstract_models import AbstractSynonym
+
+__all__ = []
```

Comment:
This should not be empty?

Reply:
It should be empty at this point. Synonym is appended on line 12